ucp/_libs/ucx_api.pyx

```diff
@@ -10,12 +10,17 @@ import weakref
 
 from posix.stdio cimport open_memstream
 
-from cpython.buffer cimport PyBUF_FORMAT, PyBUF_ND, PyBUF_WRITABLE
+from cpython.buffer cimport PyBUF_FORMAT, PyBUF_ND, PyBUF_READ, PyBUF_WRITABLE
+from cpython.memoryview cimport (
+    PyMemoryView_FromMemory,
+    PyMemoryView_FromObject,
+    PyMemoryView_GET_BUFFER,
+)
```

Comment:
Seems like none of these new imports are used anywhere, are they necessary?

Reply:
Nope, I ended up using `Array` instead of writing my own buffer handling :)